### PR TITLE
Changest to Twitter code

### PR DESCRIPTION
**Pull request by sathyendrasaran**

### CODE REVIEW

File: process_twitter_visualize/main.py [hunk 1]
```diff
@@ -29,17 +34,22 @@ def check(keyword,no_of_tweets=0):
     twitter = Twython(CONSUMER_KEY, access_token=ACCESS_TOKEN)
     tweets = []
     for status in twitter.search(q='\"{}\"'.format(keyword))['statuses']:
-        if len(tweets)<no_of_tweets:
-            text = status['text'].encode('utf-8')
+        if len(tweets) < no_of_tweets:
+            text = status['text']
```
Comment (selvathiruarul) [inline process_twitter_visualize/main.py]: encode to utf-8

File: process_twitter_visualize/main.py [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+"""
+
 import argparse
 import json
 from twython import Twython
-import os
-CONSUMER_KEY = os.environ['CONSUMER_KEY']
-CONSUMER_SECRET = os.environ['CONSUMER_SECRET']
+CONSUMER_KEY = ""
+CONSUMER_SECRET = ""
```
Comment (selvathiruarul) [inline process_twitter_visualize/main.py]: why?

File: process_twitter_visualize/main.py (hunk 1 above)
Comment (sathyendrasaran) [reply process_twitter_visualize/main.py]: Hold this for further input to word cloud

File: process_twitter_visualize/main.py (hunk 2 above)
Comment (sathyendrasaran) [reply process_twitter_visualize/main.py]: reverting code

**Pull request merged**